cmd/create/cluster/cmd.go

```diff
@@ -1277,6 +1277,16 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	// Hosted clusters will be multiAZ by definition
+	if isHostedCP {
+		multiAZ = true
+		if cmd.Flags().Changed("multi-az") {
```

Comment:
Should we validate that the user didn't set the flag `multi-az` to **false**?

Reply:
It's deprecated in both cases, so, no matter what they set, internally an hypershift cluster is considered multiAZ for the control plane.
The number of private subnets will dynamically indicate it the dataplane is singleAZ or multiAZ.

the `cmd.Flags().Changed("multi-az")` should be `true` in the presence of the parameter with any value, right?

Reply:
Yes, makes sense.